Voice Calculator/voiceCalc.py

```diff
@@ -0,0 +1,103 @@
+import os
+import pyttsx3
+import speech_recognition as sr
+import tkinter.messagebox as tmessage
+import wolframalpha
+
+from os.path import exists
+
+listener = sr.Recognizer()
+engine = pyttsx3.init()
+voices = engine.getProperty('voices')
+engine.setProperty('voice', voices[0].id)
+wolfprimeaplahe_app = str(input('Enter the API Token'))
```

Comment:
All the inputs in python are by default string. Therefore no need to apply str func. 
```suggestion
wolfprimeaplahe_app = input('Enter the API Token')
```